2025-07-03 02:32:23 -04:00: >>>>>>>>>>>>>>>>>>process.env.ENVIRONMENT: development
2025-07-03 02:32:28 -04:00: Unable to connect to the database: AccessDeniedError [SequelizeAccessDeniedError]: Access denied for user 'visitorbuk_user'@'localhost' (using password: YES)
    at ConnectionManager.connect (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:94:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async ConnectionManager._connect (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:222:24)
    at async /var/www/html/visitorbook.qa/visitorbook_backend/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:174:32
    at async ConnectionManager.getConnection (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:197:7)
    at async /var/www/html/visitorbook.qa/visitorbook_backend/node_modules/sequelize/lib/sequelize.js:305:26
    at async Sequelize.authenticate (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/sequelize/lib/sequelize.js:457:5) {
  parent: Error: Access denied for user 'visitorbuk_user'@'localhost' (using password: YES)
      at Packet.asError (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/packets/packet.js:740:17)
      at ClientHandshake.execute (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/commands/command.js:29:26)
      at Connection.handlePacket (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/base/connection.js:475:34)
      at PacketParser.onPacket (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/base/connection.js:93:12)
      at PacketParser.executeStart (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/packet_parser.js:75:16)
      at Socket.<anonymous> (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/base/connection.js:100:25)
      at Socket.emit (node:events:507:28)
      at Socket.emit (node:domain:489:12)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3) {
    code: 'ER_ACCESS_DENIED_ERROR',
    errno: 1045,
    sqlState: '28000',
    sqlMessage: "Access denied for user 'visitorbuk_user'@'localhost' (using password: YES)",
    sql: undefined
  },
  original: Error: Access denied for user 'visitorbuk_user'@'localhost' (using password: YES)
      at Packet.asError (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/packets/packet.js:740:17)
      at ClientHandshake.execute (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/commands/command.js:29:26)
      at Connection.handlePacket (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/base/connection.js:475:34)
      at PacketParser.onPacket (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/base/connection.js:93:12)
      at PacketParser.executeStart (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/packet_parser.js:75:16)
      at Socket.<anonymous> (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/base/connection.js:100:25)
      at Socket.emit (node:events:507:28)
      at Socket.emit (node:domain:489:12)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3) {
    code: 'ER_ACCESS_DENIED_ERROR',
    errno: 1045,
    sqlState: '28000',
    sqlMessage: "Access denied for user 'visitorbuk_user'@'localhost' (using password: YES)",
    sql: undefined
  }
}
2025-07-03 02:32:28 -04:00: ðŸš€ Server running at https://localhost:4001
2025-07-03 02:39:21 -04:00: >>>>>>>>>>>>>>>>>>process.env.ENVIRONMENT: development
2025-07-03 02:39:23 -04:00: ðŸš€ Server running at https://localhost:4001
2025-07-03 02:39:23 -04:00: Unable to connect to the database: AccessDeniedError [SequelizeAccessDeniedError]: Access denied for user 'visitorbuk_user'@'localhost' (using password: YES)
    at ConnectionManager.connect (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/sequelize/lib/dialects/mysql/connection-manager.js:94:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async ConnectionManager._connect (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:222:24)
    at async /var/www/html/visitorbook.qa/visitorbook_backend/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:174:32
    at async ConnectionManager.getConnection (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/sequelize/lib/dialects/abstract/connection-manager.js:197:7)
    at async /var/www/html/visitorbook.qa/visitorbook_backend/node_modules/sequelize/lib/sequelize.js:305:26
    at async Sequelize.authenticate (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/sequelize/lib/sequelize.js:457:5) {
  parent: Error: Access denied for user 'visitorbuk_user'@'localhost' (using password: YES)
      at Packet.asError (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/packets/packet.js:740:17)
      at ClientHandshake.execute (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/commands/command.js:29:26)
      at Connection.handlePacket (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/base/connection.js:475:34)
      at PacketParser.onPacket (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/base/connection.js:93:12)
      at PacketParser.executeStart (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/packet_parser.js:75:16)
      at Socket.<anonymous> (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/base/connection.js:100:25)
      at Socket.emit (node:events:507:28)
      at Socket.emit (node:domain:489:12)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3) {
    code: 'ER_ACCESS_DENIED_ERROR',
    errno: 1045,
    sqlState: '28000',
    sqlMessage: "Access denied for user 'visitorbuk_user'@'localhost' (using password: YES)",
    sql: undefined
  },
  original: Error: Access denied for user 'visitorbuk_user'@'localhost' (using password: YES)
      at Packet.asError (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/packets/packet.js:740:17)
      at ClientHandshake.execute (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/commands/command.js:29:26)
      at Connection.handlePacket (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/base/connection.js:475:34)
      at PacketParser.onPacket (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/base/connection.js:93:12)
      at PacketParser.executeStart (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/packet_parser.js:75:16)
      at Socket.<anonymous> (/var/www/html/visitorbook.qa/visitorbook_backend/node_modules/mysql2/lib/base/connection.js:100:25)
      at Socket.emit (node:events:507:28)
      at Socket.emit (node:domain:489:12)
      at addChunk (node:internal/streams/readable:559:12)
      at readableAddChunkPushByteMode (node:internal/streams/readable:510:3) {
    code: 'ER_ACCESS_DENIED_ERROR',
    errno: 1045,
    sqlState: '28000',
    sqlMessage: "Access denied for user 'visitorbuk_user'@'localhost' (using password: YES)",
    sql: undefined
  }
}
2025-07-03 02:53:10 -04:00: >>>>>>>>>>>>>>>>>>process.env.ENVIRONMENT: development
2025-07-03 02:53:12 -04:00: ðŸš€ Server running at https://localhost:4001
2025-07-03 02:53:12 -04:00: MYSQL --> Connection to DB has been established successfully.
